src/opm/output/eclipse/Summary.cpp

```diff
@@ -1279,7 +1279,8 @@ inline std::vector<Opm::Well> find_wells( const Opm::Schedule& schedule,
         return wells;
     }
 
-    case Opm::EclIO::SummaryNode::Category::Block:
+    case Opm::EclIO::SummaryNode::Category::Aquifer:       [[fallthrough]];
+    case Opm::EclIO::SummaryNode::Category::Block:         [[fallthrough]];
```

Comment:
This corrects an oversight where `[[fallthrough]]` was originally omitted from this case.